librz/core/cmd/cmd_analysis.c

```diff
@@ -8812,3 +8443,220 @@ RZ_IPI RzCmdStatus rz_analysis_class_vtable_lookup_handler(RzCore *core, int arg
 	ls_free(classes);
 	return RZ_CMD_STATUS_OK;
 }
+
+static RzList *get_xrefs(RzAnalysisBlock *block) {
+	RzListIter *iter;
+	RzAnalysisXRef *xref;
+	RzList *list = NULL;
+	size_t i;
+	for (i = 0; i < block->ninstr; i++) {
+		ut64 ia = block->addr + block->op_pos[i];
+		RzList *xrefs = rz_analysis_xrefs_get_to(block->analysis, ia);
+		rz_list_foreach (xrefs, iter, xref) {
+			if (!list) {
+				list = rz_list_newf(free);
+			}
+			rz_list_push(list, ut64_new(xref->from));
+		}
+	}
+	return list;
+}
+
+static RzList *get_calls(RzAnalysisBlock *block) {
+	RzList *list = NULL;
+	RzAnalysisOp op;
+	ut8 *data = malloc(block->size);
+	if (data) {
```

Comment:
If (!data) return....

Reply:
Also you can put `RzList *list` after the check then.